PostProcessing/Shaders/Debug/Histogram.shader

```diff
@@ -41,7 +45,11 @@ Shader "Hidden/PostProcessing/Debug/Histogram"
             o.texcoord = o.texcoord * float2(1.0, -1.0) + float2(0.0, 1.0);
         #endif
 
+        #if SHADER_API_GLES3
+        	o.maxValue = 0.0;
```

Comment:
If we're going to return empty values for debug views, why not simply disabling the features & shaders on these platforms? It would probably be better and cleaner than adding ifdefs everywhere.

That goes for `LightMeter`, `Vectorscope` and `Waveform` as well.